RELEASE.md

```diff
@@ -0,0 +1,40 @@
+# Kabanero.io Website Release Process
+
+## Overview
+The content on the Kabanero.io Website is built from four different sources:
+- UI of the Website from: https://github.com/kabanero-io/kabanero-website
+- Docs from: https://github.com/kabanero-io/docs
+- Blogs from: https://github.com/kabanero-io/blogs
+- The guides from repositories in https://github.com/kabanero-io that begin with `guide-`
+
+## Website Release
+The website has its own independent release process.  Releases are versioned using standard semantic versioning of format **MAJOR.MINOR.PATCH**.
```

Comment:
Need to add _when_ we would do a release and what number gets bumped in the release number